From david-b@pacbell.net Mon May  1 14:05:11 2006
From: David Brownell <david-b@pacbell.net>
To: Greg KH <greg@kroah.com>
Subject: Driver Core: CONFIG_DEBUG_PM covers drivers/base/power too
Date: Mon, 1 May 2006 13:58:33 -0700
Message-Id: <200605011358.33927.david-b@pacbell.net>

The drivers/base/power PM debug messages should appear when
either PM or driver model debug are enabled.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/power/Makefile |    3 +++
 1 file changed, 3 insertions(+)

--- gregkh-2.6.orig/drivers/base/power/Makefile
+++ gregkh-2.6/drivers/base/power/Makefile
@@ -4,3 +4,6 @@ obj-$(CONFIG_PM)	+= main.o suspend.o res
 ifeq ($(CONFIG_DEBUG_DRIVER),y)
 EXTRA_CFLAGS += -DDEBUG
 endif
+ifeq ($(CONFIG_PM_DEBUG),y)
+EXTRA_CFLAGS += -DDEBUG
+endif
